gulpfile.js

```diff
@@ -108,6 +108,7 @@ createTask('check-owners', 'checkOwners', 'check-owners');
 createTask('check-renovate-config','checkRenovateConfig','check-renovate-config'); // prettier-ignore
 createTask('check-sourcemaps', 'checkSourcemaps', 'check-sourcemaps');
 createTask('check-types', 'checkTypes', 'check-types');
+createTask('check-types-build-system', 'checkBuildSystemTypes','check-types-build-system'); // prettier-ignore
```

Comment:
This could be a mouthful to say / type during discussions. Maybe shorten to `check-build-system`, `checkBuildSystem`, and `check-build-system.js` respectively?

Reply:
That makes sense